utils/deploy-manager.js

```diff
@@ -7,9 +7,10 @@ const ConfiguratorLoader = require("./configurator-loader.js");
 const PrivateKeyLoader = require("./private-key-loader.js");
 const ABIUploader = require("./abi-uploader.js");
 const VersionUploader = require("./version-uploader.js");
+const utils = require("./utilities.js");
 
 const defaultConfigs = {
-  gasPrice: 20000000000, // 20 Gwei
+  gasPrice: utils.bigNumberify(process.env.DEPLOYER_GAS_PRICE || 20000000000),
```

Comment:
Can this be in config rather than env vars? We have settings->deployer configuration in there already.

Reply:
No because the value to use depends on the network status (typically the `fast` value on https://ethgasstation.info/) and we want to set it at the beginning of a deployment.